cloud/aws/main.tf

```diff
@@ -83,7 +84,7 @@ resource "null_resource" "cockroach-initializer" {
     inline = [
       "bash download_binary.sh cockroach/cockroach ${var.cockroach_sha}",
       "mkdir -p logs",
-      "./cockroach init --logtostderr=true --stores=sdd=data",
+      "./cockroach init --logtostderr=true --stores=${element(split(",", "${var.stores}"), 0)}",
```

Comment:
The flag passed to terraform will be something along the lines of: `--var=stores=ssd=data,ssd=data2`. I'm just curious whether this works, which depends on how terraform parses the `--var` flag.
If they do a `strings.Split('=')` they'll split the `ssd=data` parameter as well.
If they just consume the first `=`, then it's fine. Good to test it out though.